rgf/sklearn.py

```diff
@@ -187,17 +187,27 @@ def _validate_params(max_leaf,
 
 
 def _sparse_savetxt(filename, input_array):
+    try:
+        from itertools import izip
+        zip_func = izip
+    except ImportError:
+        zip_func = zip
+    spmatrix = input_array.tocsr().tocoo()
```

Comment:
I tried only `input_array.tocoo()` but input matrix in csr format has another structure and the algorithm didn't work (you can see commit e82dd9e0687aabb760748290478059791db17e68). As I understood, `tocoo()` allows just to work easy with three numpy-arrays but the structure of these arrays are provided by source sparse format.